src/Lemon/Containers/Converters/NodeContainer2BinaryCia.cs

```diff
@@ -77,11 +78,25 @@ public BinaryFormat Convert(NodeContainerFormat source)
             };
 
             Node root = source.Root;
+
+            var title = root.Children["title"].GetFormatAs<TitleMetadata>();
+            UpdateTitleMetadata(title, root.Children["content"]);
+
+            var binaryConverter = new TitleMetadata2Binary();
+            var binaryTitle = binaryConverter.Convert(title);
+            root.Children["title"].ChangeFormat(binaryTitle);
+
             WriteHeader(writer, root);
 
             WriteFile(writer, root.Children["certs_chain"]);
             WriteFile(writer, root.Children["ticket"]);
-            WriteFile(writer, root.Children["title"]);
+
+            writer.Endianness = EndiannessMode.BigEndian;
+            binaryTitle.Stream.WriteTo(writer.Stream);
+            writer.Endianness = EndiannessMode.LittleEndian;
+
+            writer.Write(new byte[0x1c]);
```

Comment:
I think this is needed because the padding it's missing. You need to add `writer.WritePadding(0x00, BlockSize);` like we have in `WriteFile`.